lambdas/drivingpermitcheck/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
 test {
 	useJUnitPlatform()
 	finalizedBy jacocoTestReport
+	testLogging.showStandardStreams true
```

Comment:
Is this just adding more logging for unit tests?

Reply:
This was for local testing purposes, will remove.